src/StateWorkflowServiceProvider.php

```diff
@@ -49,22 +49,40 @@ protected function configPath()
         return __DIR__.'/../config/workflow.php';
     }
 
+    /**
+     * Return migrations path.
+     *
+     * @return string
+     */
+    private function migrationPath()
+    {
+        return __DIR__.'/../database/migrations';
```

Comment:
```
database_path('migrations');
```

Reply:
hmm this will not work for package. only works on the framework

Reply:
@norbybaru @datashaman the multi tenant package i'm using moves the migration files into a folder
`database_path('migrations/tenant')` any idea how we can support these two folder structure?

Reply:
BTW I'll send in a PR for laravel 8 support, working on upgrading the Laravel version in property project